port_ocean/clients/port/mixins/organization.py

```diff
@@ -0,0 +1,31 @@
+from typing import List
+import httpx
+from loguru import logger
+from port_ocean.clients.port.authentication import PortAuthentication
+from port_ocean.clients.port.utils import handle_status_code
+
+
+class OrganizationClientMixin:
+    def __init__(
+        self,
+        auth: PortAuthentication,
+        client: httpx.AsyncClient,
+    ):
+        self.auth = auth
+        self.client = client
+
+    async def _get_organization_feature_toggles(self) -> httpx.Response:
+        logger.info("Fetching organization feature toggles")
+
+        response = await self.client.get(
+            f"{self.auth.api_url}/organization",
+            headers=await self.auth.headers(),
+        )
+        return response
+
+    async def get_organization_feature_toggles(
+        self, should_raise: bool = True, should_log: bool = True
+    ) -> List[str]:
+        response = await self._get_organization_feature_toggles()
+        handle_status_code(response, should_raise, should_log)
+        return response.json().get("organization", {}).get("featureFlags", [])
```

Comment:
I wonder if `featureFlags` is the right term here? I would prefer using `enabledFeatures` / `capabilities` / `features`

Reply:
That's the existing port api schema, not new